src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,82 @@
+Feature: Validate the reference of descriptors when creating reosurces
+
+        Background
+            Given this student exists:
+                  | Id                               | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | fafbbd2024664ee786c05e899fbfe9bc | 604824          | 2010-01-13 | Traci     | Mathews     |
+              And this person exists:
+                  | personId                         | sourceSystemDescriptor |
+                  | 0063560B5AC641DBBCCE4B121C5CA37B | State                  |
+
+
+        @ignore
+        Scenario: User CAN'T create a resource when reference doesn't exist
```

Comment:
Suggestion: "... when a descriptor doesn't exist" instead of "... when a reference doesn't exist". Using the word "reference" is technically correct, but it is even better to say that the _descriptor_ doesn't exist.

Reply:
Completed